docs/user-guide/spark-user-guide.md

```diff
@@ -243,6 +243,58 @@ For the initial pilot implementation of the Spark application, join logic is lim
 
 If the user wishes to limit which aggregation contexts are evaluated during an application run, there is a program option `-a` that can be used to specify one or more contextKey values for the aggregation contexts that should be evaluated.
 
+#### Advanced Joins
+
+When using normalized data, you might encounter a need to join on mutiple tables to create the desired context.
+
+Let's consider how we would need to set up a Context Definition to support using standardized valuesets within the OMOP data model.
+
+```plantuml
```

Comment:
This integration with PlantUML is pretty neat. I look forward to using it elsewhere.